test/rest.middleware.test.js

```diff
@@ -99,6 +99,18 @@ describe('loopback.rest', function() {
       .expect(200, done);
   });
 
+  it('allows models to provide a custom HTTP path', function(done) {
+    var ds = app.dataSource('db', { connector: loopback.Memory });
+    var CustomModel = ds.createModel('CustomModel',
+      {name: String},
+      {'http': { 'path': 'domain1/CustomModelPath' }
```

Comment:
Coding style: we put spaces inside square brackets, e.g. `{ name: String }`. Also property names (keys) don't need quotes, i.e. `{ http: {...} }`.